Example/Emission/ARRootViewController.m

```diff
@@ -358,6 +360,14 @@ - (ARCellData *)jumpToFavorites
   }];
 }
 
+- (ARCellData *)jumpToFair
+{
+  return [self tappableCellDataWithTitle:@"Fair" selection:^{
+    id viewController = [[ARFairComponentViewController alloc] initWithFairID:@"the-armory-show-2014"];
+    [self.navigationController pushViewController:viewController animated:YES];
+  }];
+}
+
```

Comment:
So that the admin app Emission can jump into the screen, emulating how it would work in Eigen